core/src/main/java/org/apache/calcite/rel/rules/FilterMultiJoinMergeRule.java

```diff
@@ -52,10 +55,24 @@ public FilterMultiJoinMergeRule(RelBuilderFactory relBuilderFactory) {
         relBuilderFactory, null);
   }
 
+  /**
+   * Creates a FilterMultiJoinMergeRule that uses a generic
+   * {@link Filter}
+   * @param filterClass filter class
+   * @param relBuilderFactory builder factory for relational expressions
+   */
+  public FilterMultiJoinMergeRule(Class<? extends Filter> filterClass,
```

Comment:
The main Javadoc of the class (around line 34) has to be adapted too.

Reply:
done